### ISSUE TITLE
Add MacStadium logo

### ISSUE THREAD

**Issue by s09bQ5** [UltraStar-Deluxe/ultrastar-deluxe.github.io#7]
MacStadium provides remote access to a Mac mini for testing and development of UltraStar Deluxe. They have asked for their Logo to be put on our landing page.
![MacStadium-developerlogo](https://user-images.githubusercontent.com/10702429/72195156-66ae2a80-3411-11ea-93ad-db77804ab493.png)

**Comment by basisbit** [comment]
sorry, but I don't think it would be appropriate to advertise for any specific company like this. There are other parties who already put much more work/resources into this project and did not get anything like this.
The free software foundation guidelines for free/libre open source projects discourage this.
If you need access to some Apple device running macOS, you could use one of those VMware images for current versions of macOS that are available at some places in the internet.

@AlexanderS @kamischi @bohning could you please vote on this?

**Comment by kamischi** [comment]
I do not think that the availability of a physical Mac resolves the actual bottle neck, when it comes to the support of macOS, in particular compared to man power. So, besides the more general ideas about free/libre open source, not much would be gained from it. If i am wrong on this, i favour donation based hardware from the user community.
